rollup.dev.config.js

```diff
@@ -0,0 +1,56 @@
+import babel from "@rollup/plugin-babel";
+import filesize from "rollup-plugin-filesize";
+import livereload from "rollup-plugin-livereload"
+import postcss from "rollup-plugin-postcss"
+import serve from "rollup-plugin-serve"
+import sourcemaps from "rollup-plugin-sourcemaps";
+
+export default commandLineArgs => {
```

Comment:
This file is 99% identical to `rollup.config.js` - it would be really helpful to not duplicate this as the project grows.  Since this version is only called with `--watch` - why not merge these and simply dispatch the `minimize` and `plugins` changes based on this flag as you do `--port`